src/Panel/View.php

```diff
@@ -266,6 +266,7 @@ public static function globals(): array
 				],
 				'debug'       => $kirby->option('debug', false),
 				'kirbytext'   => $kirby->option('panel.kirbytext', true),
+				'theme'       => $kirby->option('panel.theme', 'automatic'),
```

Comment:
I don't know why, but instead of `automatic`, how about `auto` or `system`? 🙂

Reply:
@afbora changed it.